src/main/java/com/zendesk/maxwell/schema/MysqlSchemaStore.java

```diff
@@ -119,7 +122,13 @@ public void clone(Long serverID, BinlogPosition position) throws SchemaStoreExce
 			getSchema();
 
 			MysqlSavedSchema cloned = new MysqlSavedSchema(serverID, caseSensitivity, getSchema(), position, savedSchema.getSchemaID(), empty);
-			cloned.save(c);
+			Long schemaId = cloned.save(c);
+
+			String logMessage = "clone schema @" + position + " based on id " + savedSchema.getSchemaID();
+			if (schemaId != null) {
+				logMessage += ", new schema id is " + schemaId;
+			}
+			LOGGER.info(logMessage);
```

Comment:
Could just use `String.valueOf(schemaId)` here (and above) if all we care about is avoiding NPE